<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-05-29T01:58:24 IST" name="Default suite" finished-at="2021-05-29T01:59:30 IST" duration-ms="66192">
    <groups>
    </groups>
    <test started-at="2021-05-29T01:58:24 IST" name="Default test" finished-at="2021-05-29T01:59:30 IST" duration-ms="66192">
      <class name="endTOend_raBiProject.listeners_addskill">
        <test-method is-config="true" signature="setup()[pri:0, instance:endTOend_raBiProject.listeners_addskill@4233e892]" started-at="2021-05-29T01:58:24 IST" name="setup" finished-at="2021-05-29T01:58:36 IST" duration-ms="11359" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="add_skills()[pri:0, instance:endTOend_raBiProject.listeners_addskill@4233e892]" started-at="2021-05-29T01:58:36 IST" name="add_skills" finished-at="2021-05-29T01:59:25 IST" duration-ms="49179" status="FAIL">
          <exception class="org.apache.commons.mail.EmailException">
            <message>
              <![CDATA[Sending the email to the following server failed : smtp.gmail.com:465]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.apache.commons.mail.EmailException: Sending the email to the following server failed : smtp.gmail.com:465
at org.apache.commons.mail.Email.sendMimeMessage(Email.java:1469)
at org.apache.commons.mail.Email.send(Email.java:1496)
at Methods.util.emails(util.java:59)
at endTOend_raBiProject.listeners_addskill.add_skills(listeners_addskill.java:130)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials v21sm5118616pgh.12 - gsmtp

at com.sun.mail.smtp.SMTPTransport$Authenticator.authenticate(SMTPTransport.java:932)
at com.sun.mail.smtp.SMTPTransport.authenticate(SMTPTransport.java:843)
at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:748)
at javax.mail.Service.connect(Service.java:388)
at javax.mail.Service.connect(Service.java:246)
at javax.mail.Service.connect(Service.java:195)
at javax.mail.Transport.send0(Transport.java:254)
at javax.mail.Transport.send(Transport.java:124)
at org.apache.commons.mail.Email.sendMimeMessage(Email.java:1459)
... 31 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.commons.mail.EmailException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_skills -->
        <test-method is-config="true" signature="exitup()[pri:0, instance:endTOend_raBiProject.listeners_addskill@4233e892]" started-at="2021-05-29T01:59:25 IST" name="exitup" finished-at="2021-05-29T01:59:30 IST" duration-ms="5588" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- exitup -->
      </class> <!-- endTOend_raBiProject.listeners_addskill -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
